build/automation/var/profile/dev.mk

```diff
@@ -102,11 +102,11 @@ GOOGLE_MAPS_API_KEY := $(or $(GOOGLE_MAPS_API_KEY), )
 
 
 #Once wiremock is deployed to dev environment calls to postcode api will be mocked
-POSTCODE_MAPPING_SERVICE_URL := https://uec-dos-api-sfsa-$(PROFILE)-uec-dos-api-sfs-mock-postcode-ingress.k8s-nonprod.texasplatform.uk/api
+POSTCODE_MAPPING_SERVICE_URL := https://uec-sf-sfsa-$(PROFILE)-uec-sf-sfs-mock-postcode-ingress.k8s-nonprod.texasplatform.uk/api
 POSTCODE_MAPPING_USER := fuzzy-search-api@nhs.net
 
 #Authentication login endpoint is set for fuzzy search at the moment. This should be configured to point authentication service api
-AUTH_LOGIN_URL := https://uec-dos-api-sfsa-$(PROFILE)-uec-dos-api-sfs-service.$(TEXAS_HOSTED_ZONE)
+AUTH_LOGIN_URL := https://uec-sf-sfsa-$(PROFILE)-uec-sf-sfs-service.$(TEXAS_HOSTED_ZONE)
```

Comment:
Can we use the same variable PROJECT_GROUP_SHORT instead of "uec-sf" please? we are using in other  environments